metrics/pipe_tasks.yaml

```diff
@@ -0,0 +1,29 @@
+# Alert Production metric definitions in pipe_tasks
```

Comment:
Oh I see what you mean. I guess the trick is that some metrics (especially generic ones like run times, may be shared by multiple user groups. Using tags like you are is a good way to declare usage/ownership. But it's fine if you want to keep those headers as-is for now.